Resources/doc/installation.rst

```diff
@@ -0,0 +1,60 @@
+Installation
+============
+
+Installation Using Symfony Flex
+-------------------------------
+
+`Symfony Flex`_ is the new way to manage dependencies on Symfony 3.3 and higher
```

Comment:
I would say 3.4 here, as using Flex with SF 3.3 is not entirely recommended (it does not work as good as in 3.4+). I don't want to push people to Flex on 3.3 in this bundle (they can still do it, but this doc should not be responsible for it 😄)